src/Workspaces/Core/Portable/Classification/Classifier.cs

```diff
@@ -99,7 +113,8 @@ internal static PooledObject<SegmentedList<ClassifiedSpan>> GetPooledList(out Se
             classificationService.AddSemanticClassifications(semanticModel, textSpan, getNodeClassifiers, getTokenClassifiers, semanticClassifications, options, cancellationToken);
 
             // intentionally adding to the semanticClassifications array here.
-            embeddedLanguageService.AddEmbeddedLanguageClassifications(project, semanticModel, textSpan, options, semanticClassifications, cancellationToken);
+            if (includedEmbeddedClassifications && project != null)
+                embeddedLanguageService.AddEmbeddedLanguageClassifications(services, project, semanticModel, textSpan, options, semanticClassifications, cancellationToken);
```

Comment:
it would be nifty to support embedded languages *within* an embedded language (like Regex with C#-test), but got too complex for now.